templates/practice/documentation/work-and-careers.html

```diff
@@ -0,0 +1,121 @@
+          <div class="p-side-navigation__drawer-header">
+            <button class="p-side-navigation__toggle--in-drawer js-drawer-toggle" aria-controls="drawer">
+              Toggle table of contents
+            </button>
+          </div>
+          <ul>
+            <li class="p-side-navigation__item">
+              <a class="highlight-link" href="#being-a-technical-author">Being a technical author</a>
+            </li>
+            <li class="p-side-navigation__item">
+              <a class="highlight-link" href="#in-the-documentation-team">In the documentation team</a>
+            </li>
+            <li class="p-side-navigation__item">
+              <a class="highlight-link" href="#how-we-hire">How we hire</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </aside>
+    <main class="col-7">
+      <div id="introduction">
+        <h2>Working in documentation at Canonical</h2>
+        <p>We are hiring now for <a href="/careers/3798310/technical-author-ubuntu-and-canonical-products-remote">Technical Authors</a> and <a href="/careers/4752030">Senior Technical Author</a> &mdash; individuals at various levels of experience and seniority. They will be placed in multiple different teams to work on software and products of all kinds.</p>
+        <p><a href="/careers/all?search=%22Technical+Author%22" class="p-button">Join the team</a></p>
+      </div>
+      <h3 class="question-heading" id="being-a-technical-author" style="scroll-margin-top: 4.5rem;">Being a Canonical Technical Author</h3>
+      <p>A candidate for a Canonical Technical author role:</p>
+      <div class="row">
+        <div class="col-7">
+          <ul>
+            <li>has and uses programming skills &mdash; they might not be an engineer, but programming is amongst the things they can do</li>
+            <li>creates documentation that's part of a product, whether it's a proprietary or open-source, professional or personal</li>
+            <li>engages with the software and the development process as part of their work</li>
+            <li>thinks about and puzzles over the problems of documentation </li>
+          </ul>
+        </div>
+      </div>
+      <p>If that sounds like you &mdash; maybe you're ready to join our team.</p>
+      <h3>What is a Technical Author, exactly?</h3>
+      <p>In some software organisations, the work of documentation specialists is to produce documentation. They receive explanations and descriptions of software from their engineering colleagues, and turn those into polished output. They are expert technical <em>writers</em>.</p>
+      <p>Our documentation specialists have a different role. They are technical <em>authors</em>, members of our engineering teams. They are calledTechnical Authors because they have  technical authority: they know their products in depth, and are part of the software development process.</p>
```

Comment:
```suggestion
      <p>Our documentation specialists have a different role. They are technical <em>authors</em>, members of our engineering teams. They are called Technical Authors because they have  technical authority: they know their products in depth, and are part of the software development process.</p>
```